package.json

```
@@ -35,7 +35,7 @@
     "package-setup": "node -e \"require('./build/scripts/package.ts').setup()\"",
     "package-create": "node -e \"require('./build/scripts/package.ts').create()\"",
     "pretest": "yarn compile && yarn webpack",
-    "test": "node ./out/tests/runTest.js",
+    "test": "node ./out/tests/runTest.js && sbt test",
```

```suggestion
    "test": "sbt test && node ./out/tests/runTest.js",
```

This is the only small thing I had. I believe we want `sbt test` first since some of the node tests depend on the debugger so if it fails an sbt test we should not try to run the node ones.